dispatch/BESStopWatch.h

```diff
@@ -68,7 +72,7 @@ if (BESISDEBUG((module)) || BESISDEBUG(TIMING_LOG_KEY) || BESLog::TheLog()->is_v
 #define BES_MODULE_TIMING(message) BESStopWatch commandTimer; \
     commandTimer.start(string("Module timing: ") + (message))
 #define BES_COMMAND_TIMING(message, DHI) BESStopWatch commandTimer; \
-    commandTimer.start(string("Command timing: ") + (message) + (d_dhi_ptr->data[REQUEST_ID] + " " + d_dhi_ptr->data[LOG_INFO]))
+    commandTimer.start(string("Command timing: ") + (message) + (d_dhi_ptr->data[LOG_INFO]), d_dhi_ptr)
```

Comment:
Not sure if these changes have an impact on my earlier comments about using these macros...